main.js

```diff
@@ -149,12 +149,14 @@ module.exports = function (argumentOptions) {
         connectionOptions.tls = options.tls;
     }
 
-    if (options.routes) {
-        connectionOptions.routes = options.routes;
```

Comment:
I checked the example app.js and don't see a routes object being passed in here:
https://github.com/cnnlabs/cnn-hapi/blob/develop/example/app.js

Also settings routes that way and passing it into connectionOptions at least causes an error if I remember because it violates HAPI's JOI schema restrictions.

Can you point where in the example routes is being passed in as an argument to server?